Fix images max-width issue in IE 10 and 11

maya opened the pull request:
Images were extending beyond the grid item they were in in IE 11 and below. This fixes it for IE 10 and 11.

Fixes #47.

cc: @gboone @elainekamlley

maya committed:
Add permalinks to footer logos

maya committed:
Update url to 18f's url

maya committed:
Update hero keyword

maya committed:
Add browser fix for current nav link border highlight fix for Safari and FF

Commit:
Merge branch 'master' into fix-border-highlight-browsers

Commit:
Merge branch 'master' into update-hero-img

gboone committed:
Merge branch 'master' into add-link-footer-logo

Commit:
Adds baseurl to the image paths for these images

maya committed:
Add view on GitHub link to footer

maya committed:
Fix images max-width issue in IE

maya committed:
Fix images max-width issue in IE

Commit:
Merge pull request #48 from 18F/post-fixes

Adds baseurl to the image paths for these images

Commit:
Merge branch 'master' into fix-border-highlight-browsers

Commit:
Merge branch 'master' into add-gh-link

maya committed:
Add baseurl to img

gboone committed:
Merge branch 'master' into update-hero-img

gboone committed:
Merge pull request #39 from 18F/add-link-footer-logo

Add permalinks to footer logos

gboone committed:
Merge pull request #44 from 18F/fix-border-highlight-browsers

Fix FF and Safari current nav link state

gboone committed:
Merge pull request #43 from 18F/update-hero-img

Update blog post hero keyword

gboone committed:
Merge pull request #52 from 18F/add-baseurl-img

Add baseurl to image on Press page

gboone committed:
Merge pull request #49 from 18F/add-gh-link

Add "View on GitHub" link to footer

Commit:
Attempt to speed up incremental builds

Use an environment variable and a second config file to make
incremental builds pretty quick (time decreased for a single
page to build from ~15 seconds to ~3 seconds).

Commit:
Adds build and serve commands to shortcut the fast builds locally

Commit:
Updates the readme w/ info about shorthand commands

gboone committed:
Merge pull request #53 from 18F/jk-fast-builds

Attempt to speed up incremental builds

gboone commented:
Tested in:
- [x] IE10
- [x] IE11

gboone commented:
IE 10
![new bug](https://cloud.githubusercontent.com/assets/312468/18012271/0ddb57dc-6b76-11e6-8086-c8773b0f6442.png)

In all browsers the images are now _really small._ But the original bug is fixed.

Chrome
![archive-display-bug](https://cloud.githubusercontent.com/assets/312468/18012454/c1c12240-6b76-11e6-9240-8dc05e4696ff.png)

gboone commented:
It looks like that may be specific to the `how we work` tag archive. @_@ investigating.

Here's IE 10 on the `18f` tag. Chrome looks basically the same.
![ie-10 overflow-18f-tag](https://cloud.githubusercontent.com/assets/312468/18012527/26601792-6b77-11e6-9873-4aa6a51a75af.png)

Commit:
Fixes the tiny image in tag archives problem

Commit:
Fixes the tiny image in tag archives problem

Commit:
Revert "Fixes the tiny image in tag archives problem"

This reverts commit 8ffe3c3964f65b20634488809a563ad0d10d8284.

Commit:
For real fixes the tiny images on tags/how-we-work

Commit:
Merge branch 'fix-img-ie' of github.com:18F/beta.18f.gov into fix-img-ie

Commit:
reverts again??

Apparently 1ea295d78ac297d51d5b1cbccb325cb86bf622a2 undid the earlier revert.
WTF, Git

gboone commented:
IE 10 is good.
![its-fixed](https://cloud.githubusercontent.com/assets/312468/18013944/d78f13c8-6b7d-11e6-9da5-e4519689669b.png)

gboone commented:
IE11, too
![ie-11 overflow bug](https://cloud.githubusercontent.com/assets/312468/18014042/58f835c0-6b7e-11e6-9654-02e13e77aad1.png)
